doc/revision_history.rst

```diff
@@ -6,6 +6,36 @@
 ts_ATDomeTrajectory Revision History
```

Comment:
Just a pedantic note, could you name this release notes instead of revision history?

Reply:
I already have about a dozen packages that use revision_history.rst. I would rather not change them all (or at least would rather change them all on some other ticket if we're going to change this).